healsparse/healSparseMap.py

```diff
@@ -58,7 +58,6 @@ def __init__(self, cov_map=None, cov_index_map=None, sparse_map=None, nside_spar
         if cov_index_map is not None and cov_map is not None:
             raise RuntimeError('Cannot specify both cov_index_map and cov_map')
         if cov_index_map is not None:
-            import warnings
             warnings.warn("cov_index_map deprecated", DeprecationWarning)
```

Comment:
Consider adding `stacklevel=2` here so that people have an idea what in their code is causing the deprecation warning.